docs/articles/documentation/guides/basic-guides/assert.md

```diff
@@ -0,0 +1,145 @@
+---
+layout: docs
+title: Assert
+permalink: /documentation/guides/basic-guides/assert.html
+---
+# Assertions
+
+You can use *assertions* to check if the tested page's state matches the expected state.
+
+TestCafe provides a comprehensive set of assertions that are based on the Behavior-Driven Development style (BDD-style).
+
+This topic consists of the following sections:
+
+* [Assertion Structure](#assertion-structure)
+* [Smart Assertion Query Mechanism](#smart-assertion-query-mechanism)
+
+## Assertion Structure
+
+To construct assertions, use the [test controller](../../reference/test-api/testcontroller/README.md)'s `expect` method.
+
+The `expect` method is followed by an assertion method that accepts an expected value
+and optional arguments.
+
+The following example shows the structure of the deep equality assertion:
+
+```text
+await t.expect( actual ).eql( expected, message, options );
+```
+
+The code snippet below demonstrates how to use assertions in a test:
+
+```js
+import { Selector } from 'testcafe';
+
+
+fixture `Example page`
+   .page `http://devexpress.github.io/testcafe/example/`;
+
+
+test('Check property of element', async t => {
+   const developerNameInput = Selector('#developer-name');
+
+   await t
+       .expect(developerNameInput.value).eql('', 'input is empty')
+       .typeText(developerNameInput, 'Peter Parker')
+       .expect(developerNameInput.value).contains('Peter', 'input contains text "Peter"');
+});
+```
+
+The following assertion methods are available:
+
+* [Deep Equal](../../reference/test-api/testcontroller/expect/eql.md)
+* [Not Deep Equal](../../reference/test-api/testcontroller/expect/notEql.md)
+* [Ok](../../reference/test-api/testcontroller/expect/ok.md)
+* [Not Ok](../../reference/test-api/testcontroller/expect/notOk.md)
+* [Contains](../../reference/test-api/testcontroller/expect/contains.md)
+* [Not Contains](../../reference/test-api/testcontroller/expect/notContains.md)
+* [Type of](../../reference/test-api/testcontroller/expect/typeOf.md)
+* [Not Type of](../../reference/test-api/testcontroller/expect/notTypeOf.md)
+* [Greater than](../../reference/test-api/testcontroller/expect/gt.md)
+* [Greater than or Equal to](../../reference/test-api/testcontroller/expect/gte.md)
+* [Less than](../../reference/test-api/testcontroller/expect/lt.md)
+* [Less than or Equal to](../../reference/test-api/testcontroller/expect/lte.md)
+* [Within](../../reference/test-api/testcontroller/expect/within.md)
+* [Not Within](../../reference/test-api/testcontroller/expect/notWithin.md)
+* [Match](../../reference/test-api/testcontroller/expect/match.md)
+* [Not Match](../../reference/test-api/testcontroller/expect/notMatch.md)
```

Comment:
Please rename all files using lower-case (except for `README.md`), update all links in the markdown and permalinks in the front matter.